rosdep/python.yaml

```
@@ -2915,6 +2915,7 @@ python-psycopg2:
   ubuntu:
     '*': [python-psycopg2]
     focal: [python3-psycopg2]
+    jammy: [python3-psycopg2]
```

Actually, it turns out that the `focal` entry is wrong.  All of our rosdep keys for python3 should start with `python3-`; the `python-` ones are implicitly Python 2.

So my suggestion here is to fix this key to remove the `focal` entry, and then add a new `python3-psycopg2` key with the appropriate entries:

```suggestion
  ubuntu: [python-psycopg2]
```

---

Seems to be the correct way according to the docs. I was not sure if python3 and python2 are still separated like this, because all distros using python2 are EOL.